script/helperFunctions.sh

```diff
@@ -1977,6 +1977,37 @@ function getAllNetworksArray() {
   # return ARRAY
   printf '%s\n' "${ARRAY[@]}"
 }
+
+# Function to retrieve coreFacets from global.json
+function getCoreFacetsArray() {
+  # Ensure GLOBAL_FILE_PATH is set and not empty
+  if [[ -z "$GLOBAL_FILE_PATH" ]]; then
+    echo "Error: GLOBAL_FILE_PATH is not set or empty."
+    exit 1
+  fi
+
+  local FILE="$GLOBAL_FILE_PATH"
+  local ARRAY=()
+
+  # Ensure the global file exists
+  if [[ ! -f "$FILE" ]]; then
+    echo "Error: Global configuration file $FILE not found at $FILE."
```

Comment:
Maybe you wanted to do something like this instead?

```echo "Error: Global configuration file not found at $GLOBAL_FILE_PATH."```

Also please note that we have specific functions to log error() and warning() messages in our helperFunctions.sh. 
Please use those instead.